andela/mervels-bn-frontend
# #168131535 Users should be able to add/delete a comment on Travel requests

#### What does this PR do?
ft(request-comments )implement comment feature
#### Description of Task to be completed?
- Implement view comments
- Implement add comment
- Implement Delete comment
#### How should this be manually tested?
- Checkout to the branch ft-user-role-setting-168131532
- run npm npm install
- Run npm start
- Ensure that you are signed in
- Head over the route /request/1
`Only the owner and the manager can comment on a request`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131535](https://www.pivotaltracker.com/story/show/168131535)
#### Screenshots (if appropriate)
<img width="585" alt="Screen Shot 2019-11-26 at 15 35 48" src="https://user-images.githubusercontent.com/26383328/69638000-83a9bc80-1062-11ea-8da4-af5fcddd2fab.png">

#### Questions:
N/A
